cmd/ethrex/cli.rs

```diff
@@ -410,7 +410,11 @@ pub async fn import_blocks(
             blockchain
                 .add_block(block)
                 .await
-                .inspect_err(|_| warn!("Failed to add block {number} with hash {hash:#x}",))?;
+                .inspect_err(|err| match err {
+                    // Block number 1's parent not found, the chain must not belong to the same network as the genesis file
+                    ChainError::ParentNotFound if number == 1 => warn!("The chain file is not compatible genesis file. Are you sure you selected the correct network?"),
```

Comment:
The first part is a bit weird. Maybe something like this?

```suggestion
                    ChainError::ParentNotFound if number == 1 => warn!("The chain file does not have a compatible genesis file. Are you sure you selected the correct network?"),
```

Reply:
Oops looks like I missed a `with` there, nice catch!
Should read: `The chain file is not compatible with the genesis file`.